proposals/NNNN-experiment-tracking/README.md

```diff
@@ -0,0 +1,495 @@
+familiar MLFlow SDK APIs to log metrics, parameters, and artifacts, and then view all my experiments in a unified
+Kubeflow dashboard alongside pipeline runs and other ML workflows.
+
+#### Story 2: ML Engineer Comparing Runs Across Different Sources
+
+As an ML engineer, I want to compare experiment runs that come from different sources (Jupyter notebooks, pipeline runs,
+training jobs) in a single unified interface, so that I can make informed decisions about model performance regardless
+of how the experiments were executed. I should be able to filter runs by experiment, parameters, metrics, and source,
+then visualize comparisons using parallel coordinates plots, scatter plots, and other chart types.
+
+#### Story 3: Platform Administrator Managing Multi-tenant Experiment Tracking
+
+As a platform administrator, I want to ensure that experiment tracking data is properly isolated by Kubernetes namespace
+using RBAC, so that different teams can securely share the same Kubeflow cluster without data leakage. I should be able
+to configure namespace-level access controls for experiments, runs, and artifacts, while maintaining the ability to
+provide global visibility for cross-team collaboration when needed.
+
+### Risks and Mitigations
+
+1. **Migration Challenges**: The proposal involves migrating away from MLMD (Machine Learning Metadata), which is
+   currently used by Kubeflow Pipelines. This migration process could be complex and risky, potentially breaking
+   existing workflows. To mitigate this risk, we will implement extensive testing and maintain an extended transition
+   period where Kubeflow Pipelines registers to both MLMD and Model Registry.
+1. **Dependency on External Plugin**: The proposal relies heavily on MLFlow SDK compatibility through plugins, creating
+   a dependency on MLFlow's plugin architecture. This dependency could become fragile if MLFlow changes their plugin
+   APIs or if the plugins don't work as expected. To mitigate this risk, the Kubeflow SDK will also support these APIs,
+   allowing us to move away from wrapping MLFlow without breaking user code. Furthermore, popular plugins such as from
+   [Microsoft Azure](http://docs.azure.cn/en-us/machine-learning/how-to-use-mlflow-configure-tracking) would cause
+   significant disruption for the MLFlow community if they were to change their SDK plugin stance, making such changes
+   unlikely.
+
+## Design Details
+
+### Model Registry Domain Models
+
+The expanded Model Registry will include the following domain models, heavily influenced by
+[kubeflow/model-registry#1224](https://github.com/kubeflow/model-registry/issues/1224#issuecomment-3068005968). This is
+meant to be a high-level design with further refinement being done in a Model Registry specific KEP or design document.
+
+1. **Experiments**: A group of related runs. This would replace experiments in Pipelines and could map to experiments in
```

Comment:
```suggestion
1. **Experiments**: A group of related runs. This would replace experiments in Pipelines and could potentially map to experiments in
```